src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_hmac.c

```diff
@@ -112,3 +112,12 @@ int32_t AppleCryptoNative_HmacFinal(HmacCtx* ctx, uint8_t* pbOutput)
     CCHmacFinal(&ctx->hmac, pbOutput);
     return 1;
 }
+
+int32_t AppleCryptoNative_HmacCurrent(const HmacCtx* ctx, uint8_t* pbOutput)
+{
+    if (ctx == NULL || pbOutput == NULL)
+        return 0;
+    
+    HmacCtx dup = *ctx;
+    return AppleCryptoNative_HmacFinal(&dup, pbOutput);
```

Comment:
Nit: seems like an odd choice to not have a bounds check here, implying the caller already knows the destination buffer is properly sized, _and_ to return the number of bytes written.